lib/Doctrine/DBAL/Platforms/OraclePlatform.php

```diff
@@ -975,24 +975,30 @@ public function getName()
      */
     protected function doModifyLimitQuery($query, $limit, $offset = null)
     {
-        $limit = (int) $limit;
```

Comment:
@morozov sounds reasonable. Thanks for clearing that up. I'll accept it as is now.